src/Backoffice/Flight/App/Resources/FlightResource.php

```diff
@@ -17,10 +18,10 @@ public function toArray(Request $request): array
     {
         return [
             'id' => $this->resource->id,
-            'airline_id' => $this->resource->airline_id,
+            'airline' => AirlineResource::make($this->whenLoaded('airline')),
             'departure_city' => CityResource::make($this->whenLoaded('city')),
             'arrival_city' => CityResource::make($this->whenLoaded('city')),
```

Comment:
```suggestion
            'departure_city' => CityResource::make($this->whenLoaded('departure_city')),
            'arrival_city' => CityResource::make($this->whenLoaded('arrival_city')),
```
Both the departure and arrival city use the same relationship? 🤔